[OSDOCS#17176]: Make changes as per the errors and warnings for CQA-2 (#114253)
openshift/openshift-docs: modules/security-network-multiple-pod.adoc

//
// * security/container_security/security-network.adoc

:_mod-docs-content-type: CONCEPT
[id="security-network-multiple-pod_{context}"]
= Using multiple pod networks

[role="_abstract"]

> Each running container has only one network interface by default. The Multus CNI plugin lets you create multiple CNI networks, and then attach any of those networks to your pods. In that way, you can do things such as separate private data onto a more restricted network and have multiple network interfaces on each node.

Reviewer: Too long
```suggestion
Each running container has only one network interface by default. You can use the Multus CNI plugin to create multiple CNI networks, and attach any of those networks to a pod. In that way, you can separate private data onto a more restricted network and have multiple network interfaces on each node.
```
